src/Nri/Ui/Table/V8.elm

```diff
@@ -176,11 +184,18 @@ viewLoadingRow columns alternatingRowColors index =
 
 
 viewLoadingColumn : Int -> Int -> Column data msg -> Html msg
-viewLoadingColumn rowIndex colIndex (Column _ _ width _ _ cellType _) =
-    cell cellType
-        [ css (stylesLoadingColumn rowIndex colIndex width ++ [ verticalAlign middle ] ++ loadingCellStyles)
-        ]
-        [ span [ css loadingContentStyles ] [] ]
+viewLoadingColumn rowIndex colIndex column =
+    case column of
+        Column _ _ width _ _ cellType ->
+            cell cellType
+                [ css (stylesLoadingColumn rowIndex colIndex width ++ [ verticalAlign middle ] ++ loadingCellStyles)
+                ]
+                [ span [ css loadingContentStyles ] [] ]
+
+        PlaceholderColumn width ->
+            cell DataCell
+                [ css (stylesLoadingColumn rowIndex colIndex width ++ [ verticalAlign middle ] ++ loadingCellStyles) ]
+                [ span [ css loadingContentStyles ] [] ]
```

Comment:
🎨 Shouldn't we skip the loading indicator in placeholder columns? Doesn't seem right to show a loading thingy signaling that content will load soon and then see it disappear.